src/main/java/redis/clients/jedis/csc/DefaultCache.java

```diff
@@ -0,0 +1,72 @@
+package redis.clients.jedis.csc;
+
+import java.util.Collection;
+import java.util.HashMap;
+import java.util.Map;
+
+public class DefaultCache extends AbstractCache {
+
+    protected final Map<CacheKey, CacheEntry> cache;
+    private final LRUEviction evictionPolicy;
```

Comment:
Perhaps use the interface `EvictionPolicy` to make it possible to switch policies?

Also have one (unused) constructor to enable creating the cache with a custom eviction policy?

Reply:
done